### PR TITLE
Exposed uiSchema for additional styling capabilities

### PR DESCRIPTION
**Pull request by mkotsollaris**

### CODE REVIEW

File: STYLING.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+No different to other DOM elements.
+
+## File Upload
+
+[Codepen example](https://codepen.io/adamlaki/pen/VYpewx)
+
+## Input
+
+// TODO can't be done from css
```
Comment (cuongtruong-tlo) [inline STYLING.md]: Can we show something nicer than TODO like "coming soon" ? or maybe not showing the ##Input at all ?

File: STYLING.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+Applying styling to `trulioo-react` components is no different than applying styling to HTML5 components, by utilzing CSS.
+
+## Bg-colour 
+
+`background-color` is able to apply changes to a certain DOM element. Use IDs and classnames to target a certain element in the DOM i.e: 
+
+```
+#root_TruliooFields .form-group.field.field-object:nth-child(2) {
+    background-color: lightblue;
+}
```
Comment (kateika) [inline STYLING.md]: do we support css-in-js styling?

Comment (mkotsollaris) [reply STYLING.md]: yes, technically it should be the same? I suspect something along the lines:
```
style.EmbedId`
...(style here)
`
```

File: STYLING.md (hunk 1 above)
Comment (mkotsollaris) [reply STYLING.md]: Good call 👍

File: STYLING.md (hunk 2 above)
Comment (kateika) [reply STYLING.md]: I guess we should add it to documentation

**Pull request merged**